test/components/cluster-autoscaler/cluster-autoscaler_test.go

```diff
@@ -31,12 +31,7 @@ const (
 )
 
 func TestClusterAutoscalerDeployments(t *testing.T) {
-	client, err := testutil.CreateKubeClient(t)
-	if err != nil {
-		t.Errorf("could not create Kubernetes client: %v", err)
-	}
-
-	t.Log("got kubernetes client")
+	client := testutil.CreateKubeClient(t)
```

Comment:
This needs rebase?

Reply:
Yes, thanks. I've done it already, but I didn't push the changes yet :smile: